dayone.hs

```diff
@@ -21,28 +21,28 @@
 
 -- Part  1:
 -- change next line: add parentheses to make this equal "Functional"
-word1 = reverse "lanoi" ++ "tcnuF"    -- "++"concatenates concatenates strings
+word1 = reverse ("lanoi" ++ "tcnuF")    -- "++"concatenates concatenates strings
   
 
 incr n = n + 1 -- don't change this...
 
 -- Part 2:
 -- change next line to make equal 50
-fifty = incr 7 * 7
+fifty = incr (7 * 7)
 
 
 -- Part 3:
 -- this function adds 2 to its parameter. Change it to use `incr`
 -- twice, rather than using `+ 2`
-add2 n = n + 2
+add2 n = incr (incr n)
 
 
 -- Part 4:
 -- You could also write `add2` as:
 newAdd2 = incr . incr
 --
 -- Use the 'net to find out what the "." operator is called
--- and write it here: «your answer»
+-- and write it here: it's the same as it is in regular math. if you have f.g(x), it's the same as f(g(x))
 
```

Comment:
Indeed it is!

The cool thing, though, is that f(g()) is two functions that can be used together, but f . g is a brand new function that does something specific.